AndroidManifest.xml

```diff
@@ -41,6 +42,7 @@
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
+                <category android:name="android.intent.category.LEANBACK_LAUNCHER" />

```

Comment:
also recommendation from "getting started with tv apps" link. I didn't see any reason to create a new intent-filter that would call the same action, so I added a new category here. Schema seems to support multiple category elements.